m4_sessions/1705-inning/group_seven.md

```diff
@@ -2,9 +2,46 @@
 
 ## List your group members here:Dan Olson, Ryan Spink, Max Glassie, Courtney Meyerhofer
 
-### Topic: Ethics and Technology of the Snowden Leaks
+# Ethicas and Technology of the Snowden Leaks
+> 
 
-### Outline
-#### Before class: Readings before session (short, infographic-like) about technologies revealed in Snowden documents
-#### Video: debate snippets showing both sides
-#### Discussion: Ask questions for pair and share discussion
+## Pre-Material
+Please review these resources to prepare you for our session. Think about what is important to you and why.
+
+#### What is Prism
+> https://www.youtube.com/watch?v=2O9F7Oq2_70
+
+#### Prism From Edward Snowden
+>https://www.youtube.com/watch?v=3P_0iaCgKLk
+
+#### What is XKeyScore
+> https://www.youtube.com/watch?v=3IzXkLOgVZ8
+
+#### All of Snowden's Files (Optional)
+> https://snowdenarchive.cjfe.org/greenstone/cgi-bin/library.cgi?e=p-00100-00---off-0snowden1--00-2----0-10-0---0---0direct-10----4-------0-1l--10-en-50---50-about---01-3-1-00-00--4--0--0-0-01-10-0utfZz-8-00&a=d&cl=CL6
+
+## Reading/Video (15 minutes )
+We will be watching 2 videos, 1 on each side of the argument. 
+> VIDEO LINK 1
+> VIDEO LINK 2
+> READING LINK
+
+## Reflection ( 10 Minutes )
```

Comment:
I might give a few less minutes for reflection (5) and add more to the discussion